.gitignore

```diff
@@ -37,3 +37,5 @@ scripts/github/.secrets
 
 ### Docker Compose ###
 docker-compose.override.yml
+check-env.sh
```

Comment:
Why ignore these files?

Reply:
These are only needed for Docker daemon running as root (Linux systems)